src/Storage/Storage.Management/help/New-AzDataLakeGen2Item.md

```diff
@@ -143,6 +143,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EncryptionContext
```

Comment:
(Optional) Do you want to add/update an example with this new parameter?

Reply:
Yes added